tests/test_ruby.sh

```diff
@@ -43,4 +52,24 @@ function test_rbenv {
   done
 }
 
+function test_rvm {
+
+  # Disable test if rvm (and rvm-prompt) is not installed
+  [ -z $(command -v rvm-prompt >/dev/null) ] && startSkipping
```

Comment:
This is not particularly useful. Our CI test runs, and my own machine, do not have `rvm-prompt` installed, so this test will rarely be run. Instead of mocking the output of `rvm`, mock `rvm-prompt`, so this test can run without it.

Reply:
The skipping strategy is often use in Perl distributions, so it was natural for me to go down that path. I have removed the conditional and added a basic mock, let me know what you think.